lib/service.js

```diff
@@ -143,7 +143,7 @@ Service.prototype.logs = function(opts, callback) {
   var optsf = {
     path: '/services/' + this.id + '/logs?',
     method: 'GET',
-    isStream: true,
+    isStream: (typeof(args.opts.follow) != 'undefined') ? args.opts.follow : false,
```

Comment:
This could also use the original logic from `lib/task.js` that you replaced.

Reply:
Agree, changing.